CHANGELOG

```diff
@@ -0,0 +1,45 @@
+BigQuery Web Version 1.0.1 (2021-02-XX)
```

Comment:
Just to replace `XX` here.

Reply:
Yup, left these as placeholders until all reviews were done. :)